.travis.yml

```diff
@@ -14,3 +14,11 @@ install:
   - echo y | yo rn-toolbox:eslint
   - echo y | yo rn-toolbox:jest
   - echo GeneratorTest | yo rn-toolbox:base --force
+  - cd ..
+before_script:
+  - npm prune
```

Comment:
If yarn.lock is found yarn is available in travis. Travis then runs "yarn" which prunes dependencies automatically.